README.md

```diff
@@ -209,28 +199,22 @@ Feel free to check the [issues page](https://github.com/Tekhlay/react-bookstore/
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project please give a⭐️ and share with your friends.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank @Microverse
+I would like to thank @Microverse.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
 - **[Question_1]**
 
   - [Answer_1]
```

Comment:
**[Optional]**

- [ ] It would be nice if you add some most frequently asked questions and their answers in this section to guide your viewers about your project and its uniqueness and to make your documentation readme file descriptive :+1: